e2e/tests/omnibus-sepolia.toml/BFP_Market_ETH.e2e.js

```diff
@@ -420,4 +420,115 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
     log({ newCollateral });
     assert.equal(newCollateral.toNumber(), 0);
   });
+
+  describe('Liquidation', async () => {
+    const marketId = require('../../deployments/extras.json').eth_market_id;
+    const collateralAddress = require('../../deployments/extras.json').weth_address;
+
+    it('should open a short', async () => {
+      await contractWrite({
+        wallet,
+        contract: 'BfpMarketProxy',
+        func: 'modifyCollateral',
+        args: [accountId, marketId, collateralAddress, ethers.utils.parseEther(String(500))],
+      });
+
+      const newDigest = await contractRead({
+        wallet,
+        contract: 'BfpMarketProxy',
+        func: 'getAccountDigest',
+        args: [accountId, marketId],
+      });
+      log({ newDigest });
+
+      const newDepositedWeth = newDigest.depositedCollaterals.find(
+        (c) => c.collateralAddress === collateralAddress
+      );
+      log({ newDepositedWeth });
+      const { now, blockTimestamp } = getTimes(provider);
+      const buffer = 1; // 1s
+      const diff = now - blockTimestamp;
+
+      if (diff < 0) {
+        await wait(Math.abs(diff) + buffer * 1000);
+      }
+      await commitBfpOrder({
+        wallet,
+        accountId,
+        marketId,
+        sizeDelta: -0.01,
+      });
+      await wait(1000);
+      await wait(5000);
+
+      const newPosition = await settleBfpOrder({ wallet, accountId, marketId });
+      assert.equal(newPosition.positionSize, -0.01);
+    });
+
+    it('should liquidate an underwater position', async () => {
+      let marketConfiguration = await contractRead({
+        wallet,
+        contract: 'BfpMarketProxy',
+        func: 'getMarketConfigurationById',
+        args: [marketId],
+      });
+
+      log({ marketConfiguration });
+
+      const marginDigest = await contractRead({
+        wallet,
+        contract: 'BfpMarketProxy',
+        func: 'getMarginDigest',
+        args: [accountId, marketId],
+      });
+      log({ marginDigest });
+      const collateralUsd = marginDigest.collateralUsd;
+      log({ collateralUsd });
+
+      // Sets minMarginUsd to be above the current collateral value
+      const updatedMarketConfiguration = {
+        ...marketConfiguration,
+        minMarginUsd: collateralUsd + 1,
+      };
+
+      const owner = await contractRead({
+        wallet,
+        contract: 'BfpMarketProxy',
+        func: 'owner',
+      });
+
+      await contractWrite({
+        wallet,
+        contract: 'BfpMarketProxy',
+        func: 'setMarketConfigurationById',
+        args: [
+          {
+            marketId,
+            ...updatedMarketConfiguration,
+          },
+        ],
+        impersonate: owner,
+      });
+
+      // start the liquidation process
+      const { tx, receipt } = await contractWrite({
+        wallet,
+        contract: 'BfpMarketProxy',
+        func: 'flagPosition',
+        args: [accountId, marketId],
+      });
+
+      log({ tx, receipt });
```

Comment:
tx info and receipt events are already logged in contractWrite. Not sure we need them here.

Reply:
Removing them